eq-author/src/components/routing/RoutingRule.js

```diff
@@ -130,7 +127,6 @@ const RoutingRule = ({
         <RoutingRuleDestinationSelector
           id="then"
```

Comment:
This id needs to be `rule.id` as otherwise you could have multiple destination selectors all with the same id.